modules/veteran/app/models/veteran/service/representative.rb

```diff
@@ -15,10 +15,18 @@ class Representative < ApplicationRecord
 
       validates :poa, presence: true
 
-      def self.for_user(user)
-        reps = where(first_name: user.first_name, last_name: user.last_name)
+      def as_json
```

Comment:
So I was working with this and just saw this part... Let's remove this and make it a serializer, and I don't see much reason to cut out some attributes, but I could be wrong about that.

Reply:
The reason I did that is because I dont think we should be exposing the ssn stuff or the birthdate since they are not on the public side of the OGC data.

Reply:
Ya ya that's true. Either way serializer is the more standard way

Reply:
Updated to use the serializer